.github/workflows/lint-php.yml

```diff
@@ -0,0 +1,58 @@
+name: Lint PHP
+on:
+  pull_request:
+    types: [ opened, edited, reopened, ready_for_review ]
+    paths:
+      - '**.php'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  phpcs:
+    name: Run PHP Code Sniffer
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      # User PHP 7.4 for compatibility with the WordPress codesniffer rules.
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.4'
+          coverage: none
+          tools: phpcs
+
+      - name: Get changed files
+        uses: technote-space/get-diff-action@v6
+        with:
+          SUFFIX_FILTER: .php
+
+      - name: Get Composer cache directory
+        id: composer-cache
+        run: echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
+        if: "!! env.GIT_DIFF"
+
+      - name: Cache Composer vendor directory
+        uses: actions/cache@v3
```

Comment:
This action is at version 4. We should add a `dependabot.yml` file to help us stay on top of these dependencies. See https://github.com/newfold-labs/wp-module-coming-soon/blob/main/.github/dependabot.yml

Reply:
@ramyakrishnai Let's also add a `.github/dependabot.yml` file like the one here: https://github.com/newfold-labs/wp-module-coming-soon/blob/main/.github/dependabot.yml

Reply:
My apologies, I forgot to add it. I'll add it now.